skietbaan_fe/src/scss/createcomp.css

```diff
@@ -3,14 +3,40 @@
   height: 39px;
   border-radius: 5px;
   border: none;
+  padding: 20px;
 }
 
 .comp-submit-btn{
+  background: #bbbcbd;
   width: 218px;
   height: 42px;
+  border-color: #e11e2b;
+  border-radius: 25px;
+  font-size: 16px;
+  font: helvetica;
+  font-weight: lighter;
+  margin-top: 20px;
+  color: white;
+}
+
+.comp-submit-btn-constainer{
+      padding: 20px;
```

Comment:
inconsistent padding

Reply:
It's due a design from the wireframes, the button needs to be further away from the inputs